src/encoded/types/base.py

```diff
@@ -202,6 +204,11 @@ def __ac_local_roles__(self):
         if 'lab' in properties:
             lab_submitters = 'submits_for.%s' % properties['lab']
             roles[lab_submitters] = 'role.lab_submitter'
+        if 'characterization_review' in properties:
+            reviewing_lab = properties['characterization_review'].get('reviewing_lab', 'none')
+            if reviewing_lab != 'none':
+                lab_reviewers = 'submits_for.%s' % reviewing_lab
```

Comment:
We should change prefix here, since these aren't submitting for labs. `reviewed_by.`?

Also I vote for more modern `'submits_for.{}'.format(reviewing_lab)` string formatting.

Reply:
Actually leaving these as `submits_for` makes sense so we don't have to update the authorization roles: https://github.com/ENCODE-DCC/encoded/blob/7256b20a750a9e31e4558c5bf73b2e798ff3c8bd/src/encoded/authorization.py#L54